order/pom.xml

```diff
@@ -5,7 +5,7 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.6.2</version>
+		<version>2.5.5</version>
```

Comment:
Por que precisou fazer downgrade na versão?

Reply:
No início, apresentou erro. Mas por ter tido varias tentativas de corrigir posso ter me enganado. 
Voltei pra versão anterior. E não deu erro agora